add first version of the privacy policy texts in german & english (#1)
Gamify-IT/privacy-policy: README.md

# Privacy Policy

> In this repo, a web page will be provided on which the privacy policy text is displayed in german & english.

Reviewer: ```suggestion
This repo stores the privacy policy text for Gamify-IT in german and english.
```
